src/lm.jl

```diff
@@ -206,7 +243,7 @@ loglikelihood(obj::LinearModel) = loglikelihood(obj.rr)
 
 function nullloglikelihood(obj::LinearModel)
     r = obj.rr
-    n = isempty(r.wts) ? length(r.y) : sum(r.wts)
+    n = nobs(r)
     -n/2 * (log(2π * nulldeviance(obj)/n) + 1)
 end
```

Comment:
Can't this just be `nullloglikelihood(obj::LinearModel) = nullloglikelihood(obj.rr)`?